src/main.rs

```diff
@@ -170,6 +171,16 @@ async fn main() -> Result<()> {
         let state = paths::load_state_file().await?;
         if should_update(&args, state.as_ref()).await? {
             update(&args, state.as_ref(), &install_path, download_attempts).await?;
+
+            // Launch post exec hook script if configured
+            match &cf.spotify.post_update_exec_hook {
+                Some(path) => {
+                    info!("Post update hook configured, running {}", &path);
+                    Command::new(path)
+                        .output().expect("Failed to launch post update exec hook!");
```

Comment:
I think this should be either `.context("Failed to launch post update exec hook")?;` or print a warning but start spotify regardless.

Reply:
Thanks, implemented. Now there is also a separate warning if the hook exits with a non-zero exit code. Both warnings do not panic the main thread.

Reply:
Actually saw I didn't publish this change, it should be there now :)